### ISSUE TITLE
Google Casting works on iOS simulator but not on real device

### ISSUE THREAD

**Issue by Poloniusz** [whowillcare/flutter_cast_video#13]
I add this package to a flutter project. While testing the features works fine on android and in iOS simulator. But in a real iOS device, google cast cannot find any available devices in my network while the simulator finds my tv. Apple airplay feature works fine in simulator and in real iOS devices. Any thoughts on what might be the issue?

**Comment by richarddanquah** [comment]
any solution on this issue

**Comment by song011794** [comment]
Is your physical device connected to the same network as the Chromecast? For example, are you connected to the same WiFi?

**Comment by Poloniusz** [comment]
Yes all devices are connected to same network. I use real android device it works fine. It is iOS devices that is having the issue. Both android and iOS simulators work fine and real android devices work fine as well except real iOS devices.

**Comment by whowillcare** [comment]
check this link, you will probably know what you are missing.

https://developers.google.com/cast/docs/ios_sender/permissions_and_discovery
